nmpolicy/internal/resolver/resolver.go

```diff
@@ -148,16 +150,27 @@ func (r resolver) resolveTernaryOperator(operator *ast.TernaryOperator,
 	r.currentNode = &(*operator)[2]
 	value, err := r.resolveStringOrCaptureEntryPath()
 	if err != nil {
+		r.updatePathError(err)
```

Comment:
The error "downcastig" to PathError to discover the node has to be done at https://github.com/nmstate/nmpolicy/blob/6ba34e92110dbbcb092a3aff5e706afa09306f36/nmpolicy/internal/resolver/resolver.go#L230-L238

Reply:
done